src/multivariate/dirichletmultinomial.jl

```diff
@@ -1,3 +1,24 @@
+"""
```

Comment:
Docstring is missing a type signature--see [here for examples.](https://github.com/invenia/BlueStyle#documentation)